book/08-customizing-git/sections/attributes.asc

```diff
@@ -226,13 +226,13 @@ $ git config --global filter.indent.clean indent
 $ git config --global filter.indent.smudge cat
 ----
 
-In this case, when you commit files that match `*.c`, Git will run them through the indent program before it stages them and then run them through the `cat` program before it checks them back out onto disk.
-The `cat` program does essentially nothing: it spits out the same data that it comes in.
-This combination effectively filters all C source code files through `indent` before committing.
+Sa kasong ito, kapag nag-commit ka ng mga file na tumutugma sa `*.c`, ang Git ay tatakbo sa mga ito sa pamamagitan ng indent na  programa bago ito i-stage nila at pagkatapos patakbuhin ang mga ito sa programa ng `cat` bago suriin ang mga ito pabalik sa disk.
```

Comment:
you could change ` ang Git ay tatakbo sa mga ito sa pamamagitan ng indent na  programa bago ito i-stage nila`  to something like `Idadaan ng Git ang mga ito sa indent na programa bago ito i-stage`